deploy/cdk/lib/foundation/foundation-stack.ts

```diff
@@ -114,15 +118,15 @@ export class FoundationStack extends Stack {
       lifecycleRules: [{ enabled: true, expiration: Duration.days(365 * 10), noncurrentVersionExpiration: Duration.days(1) }],
     })
 
-    new CfnOutput(this, 'ExportsOutputFnGetAttLogBucketCC3B17E8DomainName1D649D3C', {
-      exportName: `${this.stackName}:ExportsOutputFnGetAttLogBucketCC3B17E8DomainName1D649D3C`,
-      value: this.logBucket.bucketDomainName,
-    })
-
     this.logGroup = new LogGroup(this, 'SharedLogGroup', {
       retention: RetentionDays.ONE_YEAR,
     })
 
+    new CfnOutput(this, 'ExportsOutputFnGetAttSharedLogGroup74BE6F74Arn4AE42CDD', {
```

Comment:
Remove this. cdk should create this for you for any stacks that reference this stack's logGroup.logGroupArn.

Reply:
one of these should be outdated though because I was able to remove one.